### YamlMime:ManagedReference
items:
- uid: system.fabric.exception._fabric_server_authentication_failed_exception
  id: _fabric_server_authentication_failed_exception
  parent: system.fabric.exception
  children:
  - system.fabric.exception._fabric_server_authentication_failed_exception.FabricServerAuthenticationFailedException()
  - system.fabric.exception._fabric_server_authentication_failed_exception.FabricServerAuthenticationFailedException(String)
  - system.fabric.exception._fabric_server_authentication_failed_exception.FabricServerAuthenticationFailedException(String,Throwable)
  - system.fabric.exception._fabric_server_authentication_failed_exception.FabricServerAuthenticationFailedException(String,Throwable,boolean,boolean)
  - system.fabric.exception._fabric_server_authentication_failed_exception.FabricServerAuthenticationFailedException(Throwable)
  href: system.fabric.exception._fabric_server_authentication_failed_exception.yml
  langs:
  - java
  name: FabricServerAuthenticationFailedException
  nameWithType: FabricServerAuthenticationFailedException
  fullName: system.fabric.exception.FabricServerAuthenticationFailedException
  type: Class
  source:
    remote: &o0
      path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricServerAuthenticationFailedException.java
      branch: master
      repo: https://msazure.visualstudio.com/One/_git/sf-java
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricServerAuthenticationFailedException.java
    startLine: 5
  package: system.fabric.exception
  summary: <p>The exception that is thrown when server authentication failed. </p>
  syntax: &o1
    content: public class FabricServerAuthenticationFailedException extends FabricException
  inheritance:
  - java.lang.Object
  - RuntimeException
  - system.fabric.exception._fabric_exception
  inheritedMembers:
  - system.fabric.exception._fabric_exception.FabricException()
  - system.fabric.exception._fabric_exception.FabricException(FabricErrorCode)
  - system.fabric.exception._fabric_exception.FabricException(String)
  - system.fabric.exception._fabric_exception.FabricException(String,FabricErrorCode)
  - system.fabric.exception._fabric_exception.FabricException(String,Throwable)
  - system.fabric.exception._fabric_exception.FabricException(String,Throwable,FabricErrorCode)
  - system.fabric.exception._fabric_exception.FabricException(Throwable)
  - system.fabric.exception._fabric_exception.FabricException(Throwable,FabricErrorCode)
  - system.fabric.exception._fabric_exception.FabricException(String,Throwable,boolean,boolean)
  - system.fabric.exception._fabric_exception.FabricException(String,Throwable,boolean,boolean,FabricErrorCode)
  - system.fabric.exception._fabric_exception.getErrorCode()
- uid: system.fabric.exception._fabric_server_authentication_failed_exception.FabricServerAuthenticationFailedException()
  id: FabricServerAuthenticationFailedException()
  parent: system.fabric.exception._fabric_server_authentication_failed_exception
  href: system.fabric.exception._fabric_server_authentication_failed_exception.yml
  langs:
  - java
  name: FabricServerAuthenticationFailedException()
  nameWithType: FabricServerAuthenticationFailedException.FabricServerAuthenticationFailedException()
  fullName: system.fabric.exception.FabricServerAuthenticationFailedException.FabricServerAuthenticationFailedException()
  overload: system.fabric.exception._fabric_server_authentication_failed_exception.FabricServerAuthenticationFailedException*
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricServerAuthenticationFailedException.java
    startLine: 15
  package: system.fabric.exception
  summary: <p>Initializes a new instance of the <xref uid="system.fabric.exception._fabric_server_authentication_failed_exception" data-throw-if-not-resolved="false">FabricServerAuthenticationFailedException</xref> class. </p>
  syntax:
    content: public FabricServerAuthenticationFailedException()
- uid: system.fabric.exception._fabric_server_authentication_failed_exception.FabricServerAuthenticationFailedException(String)
  id: FabricServerAuthenticationFailedException(String)
  parent: system.fabric.exception._fabric_server_authentication_failed_exception
  href: system.fabric.exception._fabric_server_authentication_failed_exception.yml
  langs:
  - java
  name: FabricServerAuthenticationFailedException(String message)
  nameWithType: FabricServerAuthenticationFailedException.FabricServerAuthenticationFailedException(String message)
  fullName: system.fabric.exception.FabricServerAuthenticationFailedException.FabricServerAuthenticationFailedException(String message)
  overload: system.fabric.exception._fabric_server_authentication_failed_exception.FabricServerAuthenticationFailedException*
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricServerAuthenticationFailedException.java
    startLine: 40
  package: system.fabric.exception
  summary: >-
    <p>Initializes a new instance of the <xref uid="system.fabric.exception._fabric_server_authentication_failed_exception" data-throw-if-not-resolved="false">FabricServerAuthenticationFailedException</xref> class with a specified error message and error code.</p>

    <p></p>
  syntax:
    content: public FabricServerAuthenticationFailedException(String message)
    parameters:
    - id: message
      type: "26831127"
      description: <p>The error message that explains the reason for the exception. </p>
- uid: system.fabric.exception._fabric_server_authentication_failed_exception.FabricServerAuthenticationFailedException(String,Throwable)
  id: FabricServerAuthenticationFailedException(String,Throwable)
  parent: system.fabric.exception._fabric_server_authentication_failed_exception
  href: system.fabric.exception._fabric_server_authentication_failed_exception.yml
  langs:
  - java
  name: FabricServerAuthenticationFailedException(String message, Throwable cause)
  nameWithType: FabricServerAuthenticationFailedException.FabricServerAuthenticationFailedException(String message, Throwable cause)
  fullName: system.fabric.exception.FabricServerAuthenticationFailedException.FabricServerAuthenticationFailedException(String message, Throwable cause)
  overload: system.fabric.exception._fabric_server_authentication_failed_exception.FabricServerAuthenticationFailedException*
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricServerAuthenticationFailedException.java
    startLine: 29
  package: system.fabric.exception
  summary: >-
    <p>Initializes a new instance of the <xref uid="system.fabric.exception._fabric_server_authentication_failed_exception" data-throw-if-not-resolved="false">FabricServerAuthenticationFailedException</xref> class with a specified error message and error code.</p>

    <p></p>
  syntax:
    content: public FabricServerAuthenticationFailedException(String message, Throwable cause)
    parameters:
    - id: message
      type: "26831127"
      description: <p>The error message that explains the reason for the exception. </p>
    - id: cause
      type: a04fafd7
      description: <p>The exception that is the cause of the current exception, or a null reference if no inner exception is specified. </p>
- uid: system.fabric.exception._fabric_server_authentication_failed_exception.FabricServerAuthenticationFailedException(String,Throwable,boolean,boolean)
  id: FabricServerAuthenticationFailedException(String,Throwable,boolean,boolean)
  parent: system.fabric.exception._fabric_server_authentication_failed_exception
  href: system.fabric.exception._fabric_server_authentication_failed_exception.yml
  langs:
  - java
  name: FabricServerAuthenticationFailedException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  nameWithType: FabricServerAuthenticationFailedException.FabricServerAuthenticationFailedException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  fullName: system.fabric.exception.FabricServerAuthenticationFailedException.FabricServerAuthenticationFailedException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  overload: system.fabric.exception._fabric_server_authentication_failed_exception.FabricServerAuthenticationFailedException*
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricServerAuthenticationFailedException.java
    startLine: 48
  package: system.fabric.exception
  syntax:
    content: protected FabricServerAuthenticationFailedException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
    - id: enableSuppression
      type: 4fc6e284
    - id: writableStackTrace
      type: 4fc6e284
- uid: system.fabric.exception._fabric_server_authentication_failed_exception.FabricServerAuthenticationFailedException(Throwable)
  id: FabricServerAuthenticationFailedException(Throwable)
  parent: system.fabric.exception._fabric_server_authentication_failed_exception
  href: system.fabric.exception._fabric_server_authentication_failed_exception.yml
  langs:
  - java
  name: FabricServerAuthenticationFailedException(Throwable cause)
  nameWithType: FabricServerAuthenticationFailedException.FabricServerAuthenticationFailedException(Throwable cause)
  fullName: system.fabric.exception.FabricServerAuthenticationFailedException.FabricServerAuthenticationFailedException(Throwable cause)
  overload: system.fabric.exception._fabric_server_authentication_failed_exception.FabricServerAuthenticationFailedException*
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricServerAuthenticationFailedException.java
    startLine: 44
  package: system.fabric.exception
  syntax:
    content: public FabricServerAuthenticationFailedException(Throwable cause)
    parameters:
    - id: cause
      type: a04fafd7
references:
- uid: system.fabric.exception._fabric_server_authentication_failed_exception.FabricServerAuthenticationFailedException*
  name: FabricServerAuthenticationFailedException
  nameWithType: FabricServerAuthenticationFailedException.FabricServerAuthenticationFailedException
  fullName: system.fabric.exception.FabricServerAuthenticationFailedException.FabricServerAuthenticationFailedException
  package: system.fabric.exception
- uid: "26831127"
  spec.java:
  - name: String
    fullName: String
- uid: a04fafd7
  spec.java:
  - name: Throwable
    fullName: Throwable
- uid: 4fc6e284
  spec.java:
  - name: boolean
    fullName: boolean
- uid: system.fabric.exception._fabric_exception.FabricException()
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException()
  nameWithType: FabricException.FabricException()
  fullName: system.fabric.exception.FabricException.FabricException()
  type: Constructor
  syntax:
    content: public FabricException()
- uid: system.fabric.exception._fabric_exception.FabricException(FabricErrorCode)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(FabricErrorCode errorCode)
  nameWithType: FabricException.FabricException(FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: public FabricException(FabricErrorCode errorCode)
    parameters:
    - id: errorCode
      type: system.fabric._fabric_error_code
- uid: system.fabric.exception._fabric_exception.FabricException(String)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(String message)
  nameWithType: FabricException.FabricException(String message)
  fullName: system.fabric.exception.FabricException.FabricException(String message)
  type: Constructor
  syntax:
    content: public FabricException(String message)
    parameters:
    - id: message
      type: "26831127"
- uid: system.fabric.exception._fabric_exception.FabricException(String,FabricErrorCode)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(String message, FabricErrorCode errorCode)
  nameWithType: FabricException.FabricException(String message, FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(String message, FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: public FabricException(String message, FabricErrorCode errorCode)
    parameters:
    - id: message
      type: "26831127"
    - id: errorCode
      type: system.fabric._fabric_error_code
- uid: system.fabric.exception._fabric_exception.FabricException(String,Throwable)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(String message, Throwable cause)
  nameWithType: FabricException.FabricException(String message, Throwable cause)
  fullName: system.fabric.exception.FabricException.FabricException(String message, Throwable cause)
  type: Constructor
  syntax:
    content: public FabricException(String message, Throwable cause)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
- uid: system.fabric.exception._fabric_exception.FabricException(String,Throwable,FabricErrorCode)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(String message, Throwable cause, FabricErrorCode errorCode)
  nameWithType: FabricException.FabricException(String message, Throwable cause, FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(String message, Throwable cause, FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: public FabricException(String message, Throwable cause, FabricErrorCode errorCode)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
    - id: errorCode
      type: system.fabric._fabric_error_code
- uid: system.fabric.exception._fabric_exception.FabricException(Throwable)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(Throwable cause)
  nameWithType: FabricException.FabricException(Throwable cause)
  fullName: system.fabric.exception.FabricException.FabricException(Throwable cause)
  type: Constructor
  syntax:
    content: public FabricException(Throwable cause)
    parameters:
    - id: cause
      type: a04fafd7
- uid: system.fabric.exception._fabric_exception.FabricException(Throwable,FabricErrorCode)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(Throwable cause, FabricErrorCode errorCode)
  nameWithType: FabricException.FabricException(Throwable cause, FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(Throwable cause, FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: public FabricException(Throwable cause, FabricErrorCode errorCode)
    parameters:
    - id: cause
      type: a04fafd7
    - id: errorCode
      type: system.fabric._fabric_error_code
- uid: system.fabric.exception._fabric_exception.FabricException(String,Throwable,boolean,boolean)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  nameWithType: FabricException.FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  fullName: system.fabric.exception.FabricException.FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  type: Constructor
  syntax:
    content: protected FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
    - id: enableSuppression
      type: 4fc6e284
    - id: writableStackTrace
      type: 4fc6e284
- uid: system.fabric.exception._fabric_exception.FabricException(String,Throwable,boolean,boolean,FabricErrorCode)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace, FabricErrorCode errorCode)
  nameWithType: FabricException.FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace, FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace, FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: protected FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace, FabricErrorCode errorCode)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
    - id: enableSuppression
      type: 4fc6e284
    - id: writableStackTrace
      type: 4fc6e284
    - id: errorCode
      type: system.fabric._fabric_error_code
- uid: system.fabric.exception._fabric_exception.getErrorCode()
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: getErrorCode()
  nameWithType: FabricException.getErrorCode()
  fullName: FabricErrorCode system.fabric.exception.FabricException.getErrorCode()
  type: Method
  syntax:
    content: public FabricErrorCode getErrorCode()
    return:
      type: system.fabric._fabric_error_code
- uid: system.fabric.exception._fabric_exception
  parent: system.fabric.exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException
  nameWithType: FabricException
  fullName: system.fabric.exception.FabricException
  type: Class
  syntax:
    content: public class FabricException
- uid: system.fabric.exception._fabric_server_authentication_failed_exception
  parent: system.fabric.exception
  href: system.fabric.exception._fabric_server_authentication_failed_exception.yml
  name: FabricServerAuthenticationFailedException
  nameWithType: FabricServerAuthenticationFailedException
  fullName: system.fabric.exception.FabricServerAuthenticationFailedException
  type: Class
  summary: <p>The exception that is thrown when server authentication failed. </p>
  syntax: *o1
